habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -492,21 +480,17 @@ def _compute_actions_and_step_envs(self, buffer_index: int = 0):
         )
 
     def _collect_environment_result(self, buffer_index: int = 0):
+        assert self._nbuffers == 1
```

Comment:
So this code does not work with the double buffered sampling anymore I suppose? Or is there a way to integrate it with the double buffer sampling in the future?